docs/kubefirst/gitlab/terraform.md

```diff
@@ -0,0 +1,23 @@
+# Terraform and Atlantis
+`terraform` is our infrastructure as code layer and we manage our terraform workflows with `atlantis` automation.
+
+## Making Changes In Terraform
+
+### Automatic Plans With Atlantis
+Any merge request that includes a .tf file will prompt `atlantis` to wake up and run your terraform plan. Atlantis will post the plan's result to your merge request as a comment within a minute or so.
+
+Review and eventually approve the merge request.
+
+### Apply and Merge
+Add the comment `atlantis apply` in the approved merge request. This will prompt atlantis to wake up and run your `terraform apply`.
```

Comment:
```suggestion
Add the comment `atlantis apply` in the approved merge request. This will prompt Atlantis to wake up and run your `terraform apply`.
```